File: RA1/clase 14/JAVA/AbstractClasses/Perro.java

public class Perro extends Animal{
    private int pedigri;

    public Perro(String nombre, String especie, String raza, int pedigri){
        this.nombre = nombre;
        this.especie = especie;
        this.raza = raza;
        this.pedigri = pedigri;
    }

    public void haceSonido(){
        System.out.println(this.nombre + " ladra....");
    }
}

File: RA1/clase 14/PYTHON/Perro.py
from Animal import Animal

class Perro(Animal):
    def __init__(self, nombre, especie, raza, pedigri):
        self.nombre = nombre
        self.especie = especie
        self.raza = raza
        self.pedigri = pedigri

    def hace_sonido(self):
        print(self.nombre + " esta ladrando...")
